{
  "vulinfoid": "JVNDB-2019-007845",
  "vulinfodata": {
    "title": "Django における SQL インジェクションの脆弱性",
    "vulinfodescription": {
      "overview": "Django には、SQL インジェクションの脆弱性が存在します。"
    },
    "affected": {
      "affecteditem": [
        {
          "name": "Debian",
          "productname": "Debian GNU/Linux",
          "cpe": {
            "text": "cpe:/o:debian:debian_linux",
            "version": "2.2"
          }
        },
        {
          "name": "Django Software Foundation",
          "productname": "Django",
          "cpe": {
            "text": "cpe:/a:djangoproject:django",
            "version": "2.2"
          },
          "versionnumber": [
            "1.11.23 未満の 1.11.x",
            "2.1.11 未満の 2.1.x",
            "2.2.4 未満の 2.2.x"
          ]
        },
        {
          "name": "Fedora Project",
          "productname": "Fedora",
          "cpe": {
            "text": "cpe:/o:fedoraproject:fedora",
            "version": "2.2"
          }
        }
      ]
    },
    "impact": {
      "cvss": [
        {
          "version": "2.0",
          "severity": {
            "text": "High",
            "type": "Base"
          },
          "base": "7.5",
          "vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
        },
        {
          "version": "3.0",
          "severity": {
            "text": "Critical",
            "type": "Base"
          },
          "base": "9.8",
          "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
        }
      ],
      "impactitem": {
        "description": "情報を取得される、情報を改ざんされる、およびサービス運用妨害 (DoS) 状態にされる可能性があります。"
      }
    },
    "solution": {
      "solutionitem": {
        "description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
      }
    },
    "related": {
      "relateditem": [
        {
          "type": "vendor",
          "name": "Debian Security Advisory",
          "vulinfoid": "DSA-4498",
          "url": "https://www.debian.org/security/2019/dsa-4498"
        },
        {
          "type": "vendor",
          "name": "Django Software Foundation",
          "vulinfoid": "Archive of security issues",
          "url": "https://docs.djangoproject.com/en/dev/releases/security/"
        },
        {
          "type": "vendor",
          "name": "Django weblog",
          "vulinfoid": "Django security releases issued: 2.2.4, 2.1.11 and 1.11.23",
          "url": "https://www.djangoproject.com/weblog/2019/aug/01/security-releases/"
        },
        {
          "type": "vendor",
          "name": "Fedora Update Notification",
          "vulinfoid": "FEDORA-2019-647f74ce51",
          "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/STVX7X7IDWAH5SKE6MBMY3TEI6ZODBTK/"
        },
        {
          "type": "advisory",
          "name": "Common Vulnerabilities and Exposures (CVE)",
          "vulinfoid": "CVE-2019-14234",
          "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-14234"
        },
        {
          "type": "advisory",
          "name": "National Vulnerability Database (NVD)",
          "vulinfoid": "CVE-2019-14234",
          "url": "https://nvd.nist.gov/vuln/detail/CVE-2019-14234"
        },
        {
          "type": "cwe",
          "name": "JVNDB",
          "vulinfoid": "CWE-89",
          "url": "https://jvndb.jvn.jp/ja/cwe/CWE-89.html",
          "title": "SQLインジェクション"
        }
      ]
    },
    "history": {
      "historyitem": [
        {
          "historyno": "1",
          "datetime": "2019-08-21T16:48:03+09:00",
          "description": "[2019年08月21日]\\n  掲載"
        }
      ]
    },
    "datefirstpublished": "2019-08-21T16:48:03+09:00",
    "datelastupdated": "2019-08-21T16:48:03+09:00",
    "datepublic": "2019-08-01T00:00:00+09:00"
  }
}
